srcpkgs/mesa-asahi/template

```diff
@@ -0,0 +1,192 @@
+# Template file for 'mesa-asahi'
+pkgname=mesa-asahi
+version=20241211
+revision=1
+build_style=meson
+_llvmver=19
+_mesaver=25.0.0
+#Disable LTO flag should be present, see https://gitlab.freedesktop.org/mesa/mesa/-/issues/6911
+configure_args="-Dglvnd=enabled -Dshared-glapi=enabled -Dgbm=enabled -Degl=enabled
+ -Dosmesa=true -Dgles1=enabled -Dgles2=enabled -Dglx=dri
+ -Dlmsensors=enabled -Dplatforms=x11$(vopt_if wayland ,wayland)
+ -Dllvm=enabled -Db_lto=false -Dcpp_std=gnu++17"
+hostmakedepends="gettext flex pkg-config python3-Mako glslang llvm${_llvmver}
+ $(vopt_if wayland 'wayland-protocols wayland-devel') python3-pycparser  python3-yaml"
+makedepends="elfutils-devel expat-devel libXdamage-devel
+ libXxf86vm-devel libdrm-devel libffi-devel libva-devel
+ libvdpau-devel libxshmfence-devel ncurses-devel zlib-devel
+ $(vopt_if wayland 'wayland-devel wayland-protocols') llvm${_llvmver}-devel libsensors-devel
+ libXrandr-devel libglvnd-devel libzstd-devel libxml2-devel lua53-devel
+ libarchive-devel"
+depends="libglvnd"
+short_desc="Open source implementation of OpenGL and Vulkan - Asahi"
+maintainer="Orphaned <orphan@voidlinux.org>"
+license="MIT, LGPL-2.1-or-later"
+homepage="https://www.mesa3d.org/"
+changelog="https://docs.mesa3d.org/relnotes.html"
+distfiles="https://gitlab.freedesktop.org/asahi/mesa/-/archive/asahi-${version}/mesa-asahi-${version}.tar.gz"
+checksum=b92a8a2bf034fd29aa1f619eda241e874be1a4cbd3db0bafaba534eef6674145
+replaces="mesa>=0 mesa-libgallium>=0 libglapi>=0"
+provides="mesa-${_mesaver}_${version} mesa-libgallium-${_mesaver}_${version} libglapi-${_mesaver}_${version}"
+shlib_provides="libglapi.so libgallium-${_mesaver}-devel.so"
+
+build_helper="qemu"
+build_options="wayland"
+build_options_default="wayland"
+
+# Set subpackages manually
+subpackages="libgbm-asahi libgbm-asahi-devel libOSMesa-asahi mesa-asahi-vaapi mesa-asahi-vdpau
+ mesa-asahi-vulkan-overlay-layer mesa-asahi-dri MesaLib-asahi-devel"
+
+# Replace old mesa pkgs, superseded by libglvnd.
+replaces="libGL>=10_1<19.2.5_2 libEGL>=10_1<19.2.5_2 libGLES>=10_1<19.2.5_2"
+
+# Driver configuration
+# Check for correctness on major mesa version updates
+# Particularly, check if any new worthwhile drivers were added
+
+# swrast always present
+_gallium_drivers=" -Dgallium-drivers=swrast,asahi,virgl,zink"
+_vulkan_drivers=" -Dvulkan-drivers=swrast,virtio"
+
+configure_args+=" -Dgallium-vdpau=enabled -Dgallium-va=enabled ${_vulkan_drivers} -Dvulkan-layers=device-select,overlay"
+configure_args+=" ${_gallium_drivers}"
+makedepends+=" vulkan-loader"
+
+hostmakedepends+=" clang${_llvmver} rust rust-bindgen"
+makedepends+=" clang${_llvmver} libclc${_llvmver} rust
+	SPIRV-LLVM-Translator${_llvmver}-devel SPIRV-Tools-devel"
+subpackages+=" mesa-asahi-opencl"
+configure_args+=" -Dgallium-opencl=icd -Dgallium-rusticl=true -Drust_std=2021"
```

Comment:
none of this is conditional, just move it in with the original definition of these variables

if we don't have conditional subpackages, the `subpackages` var is not needed

Reply:
Done.